packages/cashier/src/Components/Confirm/__tests__/payment-agent-withdraw-confirm.spec.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+import PaymentAgentWithdrawConfirm from '../payment-agent-withdraw-confirm';
+import Confirm from '../../confirm';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+beforeAll(() => {
+    const portal_root = document.createElement('div');
+    portal_root.setAttribute('id', 'modal_root');
+    document.body.appendChild(portal_root);
+});
+
+afterEach(cleanup);
+
+describe('<PaymentAgentWithdrawConfirm />', () => {
+    const amount = 10;
+    const currency = 'testCurrency';
+    const description = 'testDescription';
+    const payment_agent_name = 'testName';
+    const error = {
+        code: 'testCode',
+        message: 'testMessage',
+    };
+    const requestPaymentAgentWithdraw = jest.fn();
+    const setIsTryWithdrawSuccessful = jest.fn();
+    const header = 'Please confirm the transaction details in order to complete the withdrawal:';
+
+    const data = [
+        {
+            label: 'testText',
+            value: payment_agent_name,
+            key: 'testKey',
+        },
+    ];
+
+    it('component should be rendered', () => {
+        const { container } = render(<PaymentAgentWithdrawConfirm />);
+
+        expect(container.querySelector('.cashier__wrapper--confirm')).toBeInTheDocument();
+    });
+
+    it('component <Row /> should be rendered when has data', () => {
+        const { container } = render(
+            <PaymentAgentWithdrawConfirm>
+                <Confirm data={data} />
+            </PaymentAgentWithdrawConfirm>
+        );
+
+        expect(container.querySelector('.confirm__row')).toBeInTheDocument();
+    });
+
+    it('component <FormError /> should be rendered when has an error', () => {
+        render(
+            <PaymentAgentWithdrawConfirm error={error}>
+                <Confirm data={data} error={error} />
+            </PaymentAgentWithdrawConfirm>
+        );
+
+        expect(screen.getByText('testMessage')).toBeInTheDocument();
+    });
+
+    it('header should be rendered', () => {
+        render(
+            <PaymentAgentWithdrawConfirm>
+                <Confirm header={header} />
+            </PaymentAgentWithdrawConfirm>
+        );
+
+        expect(screen.getByText(header)).toBeInTheDocument();
+    });
+
+    it(`setIsTryWithdrawSuccessful func should be triggered when click on 'Back' button`, () => {
+        render(
+            <PaymentAgentWithdrawConfirm setIsTryWithdrawSuccessful={setIsTryWithdrawSuccessful}>
+                <Confirm
+                    onClickBack={() => {
+                        setIsTryWithdrawSuccessful(false);
```

Comment:
@artsemkavaleu-deriv I don't think there is a need to this input for the mock function.

Reply:
RIght, removed extra variables and inputs, thx for notice!